misc/visual_inspector_dump.py

```diff
@@ -352,8 +368,16 @@ def generateCSV(dsid=None, url="http://ip/instance-name", filename="output.csv",
         if (not showall) and (row['InspectionType'] == "Collected"):
           # NOTE Special behavior here. We skip this row since we're filtering out training data
           continue
-        if int(row['Timestamp']) < newerthan:
+        if (row['Timestamp'] is None):
+          logging.debug("WARN: No timestamp for %s" % rows[0]['URL'])
+          continue
+        if (row['Timestamp'].strip() == ""):
+          logging.debug("WARN: No timestamp for %s" % rows[0]['URL'])
+          continue
```

Comment:
This code was sent by the devs working on the dashboard that calls this script.  looks like there could be times that a `TimeStamp` value is not present.

Note, i did ask about the use of `rows[0]['URL']` instead of `row['URL']`.  They wanted it this way, so I left it as is.